cmd/sidecar/main.go

```diff
@@ -161,7 +173,13 @@ func backupHandler(w http.ResponseWriter, req *http.Request) {
 	xbcloud := exec.Command("xbcloud", xbcloudArgs(backupConf)...)
 	xbcloud.Stdin = xbOut
 
-	log.Info("Backup starting", "destination", backupConf.Destination, "storage", backupConf.Storage.Type)
+	log.Info(
```

Comment:
please correct massage:
`2022-06-16T20:17:03.601Z	INFO	sidecar	Backup starting	{"namespace": "default", "name": "backup1", "destination": "cluster1-2022-06-16-20:16:58-full", "storage": "s3", "xtrabackupCmd": "/usr/bin/xtrabackup --backup --stream=xbstream --user=xtrabackup ", "xbcloudCmd": "/usr/bin/xbcloud put --md5 --parallel=10 --curl-retriable-errors=7 --storage=s3 --s3-bucket=operator-testing --s3-region=us-east-1 --s3-endpoint=   cluster1-2022-06-16-20:16:58-full"}`

and we do not have in  logfile (I can see in in backup pod)

Reply:
what should I correct in the message?

although I implemented it, there is the log in xtrabackup output itself:

```
$ kubectl logs xb-backup5-s3-west-722qs | grep 'recognized client arguments'
2022-06-16T14:09:35.939673-00:00 0 [Note] [MY-011825] [Xtrabackup] recognized client arguments: --backup=1 --stream=xbstream --user=xtrabackup --password=*
```